Themes/default/scripts/script.js

```diff
@@ -1443,7 +1443,7 @@ function smc_saveEntities(sFormName, aElementNames, sMask)
 	for (var i = 0, n = aElementNames.length; i < n; i++)
 	{
 		if (aElementNames[i] in document.forms[sFormName])
-			document.forms[sFormName][aElementNames[i]].value = document.forms[sFormName][aElementNames[i]].value.replace(/&#/g, '&#38;#');
+			document.forms[sFormName][aElementNames[i]].value = document.forms[sFormName][aElementNames[i]].value;
```

Comment:
I know i recommend this "solution",
when i look deeper in this code than this code doesn't do any thing more? --> can we drop this?

Reply:
Yes. That's what 4c50dbe does.